specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimdeployment.json

```diff
@@ -981,7 +981,8 @@
           "enum": [
             "Developer",
             "Standard",
-            "Premium"
+            "Premium",
+            "Basic"
```

Comment:
Has SDKs for this been published already?

Reply:
not yet.

Reply:
oh, good.